lstests/doc/README.md

```diff
@@ -0,0 +1,688 @@
+```
+
+will write down a line such as
+
+```
+Some debug information.
+```
+
+in the files `debug.log` and `trace.log`, because the log level of this line, which is `DEBUG`, is at or above the thresholds of the log files.
+
+However, this line will not show up on the screen, because `DEBUG` is lower than `INFO`, the threshold of the `ScreenWriter`.
+
+
+#### Writer Settings
+
+In general, for a writer, the settings include
+
+1. `threshold` : Information at or above the `threshold` will be written to the logging destination.
+2. `monochrome` : If `True`, remove all the style and color codes (e.g. `\033[1;31m` for bold red) when writing the text to the logging destination. Default value is `False`.
+3. `buffered` : If `True`, it will be *possible* to buffer the logging information and flush it in a coordinated manner. This is helpful when there are multiple threads running simultaneously.
+4. `show_clock_time` : Show the clock time in the line. Default value is `True`.
+5. `show_elapsed_time` : Show the elapsed time in the line. Default value is `True`.
+6. `show_filename` : Show the name of the file that invokes the logging. Default value is `True`.
+7. `show_lineno` : Show the line number of the function that invokes the logging. Default value is `True`.
+8. `show_function` : Show the name of the function that invokes the logging. Default value is `True`.
+9. `show_thread` : Show the name of the thread that invokes the logging. Default value is `True`.
+10. `show_log_level` : Show the log level of the line. Default value is `True`.
+
+##### More about Buffering
+
+Buffering only matters when there are multiple threads, for example, when creating multiple accounts in parallel, or when verifying a transaction asynchronously.
+
+For a writer, if `buffered=True`, it will be possible to keep information organized and make the log result human-readable. If `buffered=False`, logging will take place as soon as a line is ready. Multiple threads may log in an interleaving manner, unaware of others' existence. In general, setting `buffered=False` will *not* reduce the entire logging time, but will keep the lines in a strict chronological order, though the information itself may not be human-readable.
+
+Note that, `buffered=True` only provides a possibility to buffer. Buffering does not happen automatically. A line of information will only be buffered when it is explicitly told  to do so.
+
+For example, given two threads, both scheduled to execute
+
+```python
+# clus is a Cluster object
+clus.log("{}: first half.".format(thread_name), level="debug", buffer=True)
+clus.log("{}: second half.".format(thread_name), level="debug", buffer=True)
+clus.flush()
+```
+
+or equivalently
+
+```python
+# clus is a Cluster object
+clus.log("{}: first half.".format(thread_name), level="debug", buffer=True)
+clus.log("{}: second half.".format(thread_name), level="debug")
+```
+
+setting `buffered=True` will guarantee that the log file will be looking like
```

Comment:
"... that the log file will either look like"

Reply:
Fixed. Thank you.